tests/model_fields/models.py

```diff
@@ -218,6 +218,10 @@ class DataModel(models.Model):
 class Document(models.Model):
     myfile = models.FileField(upload_to='unused')
 
+
+class UniqueDocument(models.Model):
+    myfile = models.FileField(upload_to='unused', unique=True)
```

Comment:
I would modify the existing model unless that doesn't work for some reason (we try to minimize the number of models since each additional model increases the runtime of the test suite).